staging/src/k8s.io/client-go/testing/actions.go

```diff
@@ -324,6 +324,7 @@ type Action interface {
 	GetResource() schema.GroupVersionResource
 	GetSubresource() string
 	Matches(verb, resource string) bool
+	DeepCopy() Action
```

Comment:
Given that you run into an issue due to a mutation, maybe you could add some godoc on this method explaining why/when to use it?

Reply:
> Given that you run into an issue due to a mutation, maybe you could add some godoc on this method explaining why/when to use it?

Sure, but it is called by the invoking function.  Users should never see it.

Reply:
> Given that you run into an issue due to a mutation, maybe you could add some godoc on this method explaining why/when to use it?

done